packages/client/tsconfig.json

```diff
@@ -8,8 +8,9 @@
     "jsx": "react-native",
     "paths": {
       "@components/*": ["components/*"],
+      "@assets/*": ["assets/*"],
```

Comment:
Why make this change and not use it everywhere?

Reply:
I think I did use, let me check the areas that I haven't used it in.